app/controllers/api/databases.php

```diff
@@ -1440,6 +1579,394 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
         $response->dynamic($attribute, $model);
     });
 
+App::patch('/v1/databases/:databaseId/collections/:collectionId/attributes/:key/string')
```

Comment:
Do you think it would make more sense if the paths were `/type/key` instead of `/key/type`

E.g.
`v1/databases/marvel/collections/movies/attributes/string/name` instead of
`v1/databases/marvel/collections/movies/attributes/name/string`

Reply:
I think you're right 🤔